### PR TITLE
merge node-client handlers: Maid 1272

### PR DESCRIPTION
**Pull request by mmoadeli**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/542)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/routing_membrane.rs [hunk 1]
```diff
@@ -993,7 +997,7 @@ impl<F> RoutingMembrane<F> where F: Interface {
             },
             Err(InterfaceError::Abort) => {},
             Err(InterfaceError::Response(error)) => {
-                let orig_request = match resolved.get_orig_message() {
+                let _ = match resolved.get_orig_message() {
```
Comment (inetic) [inline src/routing_membrane.rs]: Mahmoud, if it's not used, can you remove the piece of code?

Comment (mmoadeli) [inline src/routing_membrane.rs]: Removed Peter

File: examples/simple_key_value_store.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@
 #![deny(deprecated, drop_with_repr_extern, improper_ctypes, missing_docs,
         non_shorthand_field_patterns, overflowing_literals, plugin_as_library,
         private_no_mangle_fns, private_no_mangle_statics, raw_pointer_derive, stable_features,
-        unconditional_recursion, unknown_lints, unsafe_code, unsigned_negation, unused,
+        unconditional_recursion, unknown_lints, unsafe_code, unused,
```
Comment (benjaminbollen) [inline examples/simple_key_value_store.rs]: is it simply removed, unsigned_negation??

Comment (mmoadeli) [inline examples/simple_key_value_store.rs]: `warning: lint unsigned_negation has been removed: replaced by negate_unsigned feature gate`

**Pull request merged**